src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMean.cpp

```diff
@@ -0,0 +1,274 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Core/Error.hpp>
+#include <OpenSpaceToolkit/Core/Types/Size.hpp>
+#include <OpenSpaceToolkit/Core/Utilities.hpp>
+
+#include <OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler/BrouwerLyddaneMean.hpp>
+
+namespace ostk
+{
+namespace astro
+{
+namespace trajectory
+{
+namespace orbit
+{
+namespace models
+{
+namespace kepler
+{
+
+using ostk::core::types::Size;
+using ostk::core::types::Integer;
+
+Angle BrouwerLyddaneMean::getMeanAnomaly() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("BrouwerLyddaneMean");
+    }
+
+    return anomaly_;
+}
+
+Angle BrouwerLyddaneMean::getTrueAnomaly() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("BrouwerLyddaneMean");
+    }
+
+    return COE::TrueAnomalyFromMeanAnomaly(anomaly_, eccentricity_, 1e-15);
+}
+
+Angle BrouwerLyddaneMean::getEccentricAnomaly() const
+{
+    if (!this->isDefined())
+    {
+        throw ostk::core::error::runtime::Undefined("BrouwerLyddaneMean");
+    }
+
+    return COE::EccentricAnomalyFromMeanAnomaly(anomaly_, eccentricity_, 1e-15);
+}
+
+COE::CartesianState BrouwerLyddaneMean::getCartesianState(
+    const Derived &aGravitationalParameter, const Shared<const Frame> &aFrameSPtr
+) const
+{
+    const COE coe = this->toCOE();
+    return coe.getCartesianState(aGravitationalParameter, aFrameSPtr);
+}
+
+BrouwerLyddaneMean::BrouwerLyddaneMean(
+    const Length &aSemiMajorAxis,
+    const Real &anEccentricity,
+    const Angle &anInclination,
+    const Angle &aRaan,
+    const Angle &anAop,
+    const Angle &aMeanAnomaly
+)
+    : COE(aSemiMajorAxis, anEccentricity, anInclination, aRaan, anAop, aMeanAnomaly, COE::AnomalyType::Mean)
+{
+}
+
+Vector6d BrouwerLyddaneMean::Cartesian(
+    const COE::CartesianState &aCartesianState,
+    const Derived &aGravitationalParameter,
+    std::function<Vector6d(const Vector6d &)> toCOEVector
+)
+{
+    const COE coe = COE::Cartesian(aCartesianState, aGravitationalParameter);
+
+    bool possibleInaccuracyLongWritten = false;
+    bool inaccuracyCriticalAngleWritten = false;
+
+    if ((coe.getEccentricity() > 0.99) || (coe.getEccentricity() < 0.0))
+    {
+        throw ostk::core::error::RuntimeError(
+            String::Format("BrouwerLyddaneMean is applicable only if 0.0 < Ecc: [{}] < 0.99.", coe.getEccentricity())
```

Comment:
```suggestion
            "BrouwerLyddaneMean is applicable only if 0.0 < Ecc: [{}] < 0.99.", coe.getEccentricity()
```
nit: This should work. Runtime error has auto formatting